### PR TITLE
Last round of master v5 Releases

### PR DESCRIPTION
**Pull request by holgerd77**
Last round of `master` v5 releases according to #1926 before `develop` gets merged into `master`. The `master` state directly before will then be branched off as `v5-maintenance`.

Open for review. 🙂 

Feel free to directly push corrections and additions.

### CODE REVIEW

**Comment by holgerd77** [comment]
(added some selected reviewers which would fit, but generally all reviewers welcome of course!)

File: packages/client/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,31 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+
+- Various `eth_` methods are now available on the engine API endpoint as of Kiln spec v2.1, PR [#1855](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1855)
+- Small engine API updates and fixes, PR [#1902](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1902)
+- Subsume engine's `INVALID_TERMINAL_BLOCK` into `INVALID` response, PR [#1919](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1919)
+
+### Merge: Optimistic (Beacon) Sync
+
+The client now supports [optimistic](https://github.com/ethereum/consensus-specs/blob/dev/sync/optimistic.md) (Beacon) sync where blocks are downloaded in reverse from the announced trusted head. This allows for a practically viable sync process in combination with a consensus client when running the client in a execution-consensus-client-combination setup on a Merge test network.
```
Comment (ryanio) [inline packages/client/CHANGELOG.md:21]:
```suggestion
The client now supports [optimistic](https://github.com/ethereum/consensus-specs/blob/dev/sync/optimistic.md) (Beacon) sync where blocks are downloaded in reverse from the announced trusted head. This allows for a practically viable sync process in combination with a consensus client when running the client in an execution-consensus-client-combination setup on a Merge test network.
```

**Review by ryanio — APPROVED** [review]
lgtm!

**Comment by holgerd77** [comment]
Nice, thanks guys! 🙂

**Comment by holgerd77** [comment]
Just published the following releases on npm:

- `ethereumjs-util` [v7.1.5](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/ethereumjs-util%407.1.5)
- `@ethereumjs/tx` [v3.5.2](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Ftx%403.5.2)
- `@ethereumjs/vm` [v5.9.1](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fvm%405.9.1)
- `@ethereumjs/client` [v0.5.0](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fclient%400.5.0)

**Pull request merged**